.github/workflows/retest.yaml

```diff
@@ -0,0 +1,24 @@
+name: Detect and Trigger Retest
+on:
+  issue_comment:
+    types: [created, edited]
+
+permissions:
+  contents: read
+
+jobs:
+  retest:
+    if: github.event.comment.author_association == 'MEMBER' && github.event.comment.body == '/retest'
+    permissions:
+      actions: write # for re-running failed jobs: https://docs.github.com/en/rest/actions/workflow-runs?apiVersion=2022-11-28#re-run-a-job-from-a-workflow-run
+    runs-on: ubuntu-latest
+    steps:
+      - name: Re-run failed jobs for this PR
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          REPO: ${{ github.repository }}
+          PR_NUMBER: ${{ github.event.issue.number }}
+        run: |
+          SHA_REF=$(gh api "/repos/$REPO/pulls/$PR_NUMBER/commits" | jq -r '.[].sha' | tail -n 1)
+          RUN_ID=$(gh api "repos/$REPO/actions/workflows/ci-build.yaml/runs?per_page=1&event=pull_request&head_sha=$SHA_REF" | jq -r '.workflow_runs[] | .id)
```

Comment:
oh I think there's a typo here, missing the closing `'` in the `jq`

Reply:
Looks like it was accidentally dropped in [this change/amendment](https://github.com/argoproj/argo-workflows/compare/9d2ace5524d0b90f8f1673f337372b5d9dfdb525..d75768955d00455d88b5f1a2e6c08ed7bdf3e618)

Reply:
Nice catch was when i removed the `head -n 1`

Reply:
Fixed in #13007